modules/ROOT/pages/validate-4-task.adoc

```diff
@@ -1,34 +1,57 @@
-= Validate Query Parameters and Headers Using APIkit
+= Validating Query Parameters and Headers
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-partial:
 
-This process assumes that you have generated flows using APIkit in Studio.
-
-You can configure APIkit Router to validate headers and query parameters to limit both these attributes to those defined in your API specification:
+Configure *APIkit Router* to enforce validation on headers and query parameters, restricting them only to the attributes specified within your API specification.
 
 // tag::process[]
-. In your Studio canvas, select *APIkit Router*.
+. In Studio, select *APIkit Router*.
 +
-image::apikit-router-selection.png["APIkit router highlighted."]
-. Select the *Edit* button for your router configuration
+image::apikit-router-selection.png["APIkit router highlighted"]
+. Click *Edit* for your router configuration.
 +
-image::router-config.png["Edit button highlighted next to the plus button, in the APIkit router tab."]
+image::router-config.png["Edit button highlighted in the APIkit router tab"]
 // end::process[]
-. Select *Query parameters Strict Validations*  to limit query parameters to those defined in your API specification, or *Headers Strict Validations* to limit headers to those defined in your API specification:
+
+. Select *Query Parameters Strict Validations* to restrict the usage of query parameters to only the ones defined in your API specification. 
+. Select *Headers Strict Validations* to limit the headers to only the ones specified in your API specification.
 +
-image::enable-verifications.png["Query parameters strict validations and headers strict validations highlighted in the router configuration tab."]
+image::enable-verifications.png["*Query parameters Strict Validations* and *Headers Strict Validations* highlighted in the router configuration tab"]
+
+If not, update the APIkit configuration code as follows:
```

Comment:
If not what?

Reply:
requesting assistance from SMEs.
Thanks!

Reply:
There are two ways to achieve the same result: following the mentioned steps in Studio UI or changing the XML code.
Sentences were rewritten. Thanks!